src/ape/cli/choices.py

```diff
@@ -27,6 +27,7 @@ class Alias(click.Choice):
 
     name = "alias"
 
+    # noinspection PyMissingConstructor
```

Comment:
What do these do?

Reply:
ah they just make Pycharm not complain. I could get rid of these if you want and just turn off inspection locally but it might also be nice for other Pycharm users

Reply:
turns out Pycharm also supports ` # noqa` expressions to do the same thing, so I did that since it is more IDE-generic